### ISSUE TITLE
Troubleshooting the install guide

### ISSUE THREAD

**Issue by pt-arvind** [code-corps/code-corps-api#240]
When I was installing from docker, I ran these two commands:

```
$ docker-compose build
$ docker-compose up
```

and received an error output that looked like this:

```
web_1            | Unchecked dependencies for environment dev:
web_1            | * httpoison (Hex package)
web_1            |   lock mismatch: the dependency is out of date (run "mix deps.get" to fetch locked version)
web_1            | ** (Mix) Can't continue due to errors on dependencies
codecorpsapi_test_1 exited with code 1
codecorpsapi_web_1 exited with code 1
```

I didn't really think too much of it until the subsequent step failed:

First off, needed to fetch/install the dependencies:

```
$ docker-compose run web mix run priv/repo/seeds.exs
Unchecked dependencies for environment dev:
* httpoison (Hex package)
  lock mismatch: the dependency is out of date (run "mix deps.get" to fetch locked version)
** (Mix) Can't continue due to errors on dependencies

$ docker-compose run web mix deps.get
...
```

Then I tried again and got a different error:

```
$ docker-compose run web mix run priv/repo/seeds.exs
...
==> arc
could not compile dependency :arc, "mix compile" failed. You can recompile this dependency with "mix deps.compile arc", update it with "mix deps.update arc" or clean it with "mix deps.clean arc"
==> code_corps
** (Mix) Failed to open Hex registry (file does not exist, run `mix hex.info` to fetch it)
```

To fix it, I ran this:

```
$ docker-compose run web mix deps.clean arc
$ docker-compose run web mix deps.update arc
```

I tried running:

```
$ docker-compose run web mix deps.compile arc
```

but this still gave me an error. I was pretty flummoxed, but came back to it this morning and tried 

```
$ docker-compose build
$ docker-compose up
```

and everything just worked. 

**TL;DR:**
- [ ] Add the get dependencies stage to the directions

```
$ docker-compose run web mix deps.get
```
- [ ] Add troubleshooting guide if someone sees similar issues. For now, I'm just gonna put it in the README.mdown, @JoshSmith let me know if that makes the guide unintelligible or busy
